src/azureExtensions.ts

```diff
@@ -0,0 +1,118 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ResourceManagementModels } from "@azure/arm-resources";
+import { localize } from "./utils/localize";
+
+export const azureExtensions: IAzExtMetadata[] = [
+    {
+        name: 'vscode-azurefunctions',
+        label: 'Functions',
+        resourceTypes: [
+            {
+                name: 'microsoft.web/sites',
+                matchesResource: isFunctionApp
+            }
+        ],
+        tutorial: {
+            label: localize('deployServerless', 'Create and deploy a serverless app'),
+            url: 'https://aka.ms/AAb5xpj'
+        },
+        reportIssueCommandId: 'azureFunctions.reportIssue'
+    },
+    {
+        name: 'vscode-azureappservice',
+        label: 'App Service',
+        resourceTypes: [
+            {
+                name: 'microsoft.web/sites',
+                matchesResource: r => !isFunctionApp(r)
+            }
+        ],
+        tutorial: {
+            label: localize('deployWebApp', 'Deploy a web app'),
+            url: 'https://aka.ms/AAb5dz2'
+        },
+        reportIssueCommandId: 'appService.ReportIssue'
+    },
+    {
+        name: 'vscode-azurestaticwebapps',
+        label: 'Static Web Apps',
+        resourceTypes: [
+            'microsoft.web/staticsites'
+        ],
+        tutorial: {
+            label: localize('deployStatic', 'Deploy a static app'),
+            url: 'https://aka.ms/AAb5xp8'
+        },
+        reportIssueCommandId: 'staticWebApps.reportIssue'
+    },
+    {
+        name: 'vscode-azureresourcegroups',
+        label: 'Resource Groups',
+        resourceTypes: [],
+        reportIssueCommandId: 'azureResourceGroups.reportIssue'
+    },
+    {
+        name: 'vscode-azurestorage',
+        label: 'Storage',
+        resourceTypes: [
+            'microsoft.storage/storageaccounts'
+        ],
+        reportIssueCommandId: 'azureStorage.reportIssue'
+    },
+    {
+        name: 'vscode-azurevirtualmachines',
+        label: 'Virtual Machines',
+        resourceTypes: [
+            'microsoft.compute/virtualmachines'
+        ],
+        reportIssueCommandId: 'azureVirtualMachines.reportIssue'
+    },
+    {
+        name: 'vscode-azureeventgrid',
+        label: 'Event Grid',
+        resourceTypes: [
+            'microsoft.eventgrid/eventsubscriptions',
+            'microsoft.eventgrid/topics'
+        ]
```

Comment:
lol don't even offer them a report issue for Event Grid, huh

Reply:
Yeah...  that extension is on an old version of the ui package that doesn't support the report issue command. Just not worth it at this point